tools/rna_tools/pipmir/tool_data_table_conf.xml.sample

```diff
@@ -0,0 +1,7 @@
+<!-- Use the file tool_data_table_conf.xml.oldlocstyle if you don't want to update your loc files as changed in revision 4550:535d276c92bc-->
```

Comment:
This two vs four columns seems to be matter. so far. I am not successful with the two columns.